var/spack/repos/builtin/packages/py-tensorflow/package.py

```diff
@@ -279,6 +284,21 @@ class PyTensorflow(Package, CudaPackage):
     #            type=('build', 'run'), when='@2.8:')
```

Comment:
Can remove the `+rocm` variant above since it's automatically added by `ROCmPackage` now